server/src/main/java/org/opensearch/index/shard/RemoteStoreRefreshListener.java

```diff
@@ -385,14 +387,16 @@ private void deleteStaleCommits() {
      * Updates the last refresh time and refresh seq no which is seen by local store.
      */
     private void updateLocalRefreshTimeAndSeqNo() {
-        segmentTracker.updateLocalRefreshTimeMs(System.nanoTime() / 1_000_000L);
+        segmentTracker.updateLocalRefreshClockTimeMs(System.currentTimeMillis());
+        segmentTracker.updateLocalRefreshTimeMs(System.nanoTime());
```

Comment:
Is the change from 
```
segmentTracker.updateLocalRefreshTimeMs(System.nanoTime() / 1_000_000L);
```

to
```
segmentTracker.updateLocalRefreshTimeMs(System.nanoTime());
```

intentional?

Reply:
Thanks for catching this. its not intentional, will update